### ISSUE TITLE
'str' object has no attribute 'spit'

### ISSUE THREAD

**Issue by gorgiaxx** [fkie-cad/cwe_checker#24]
cwe_checker/cwe_checker_to_ida/CweCheckerParser.py

line 13

"CWE-782" should be "CWE782"

line 127

```
    @staticmethod
    def _parse_cwe782(warning):
        warning.address = warning.warning.spit("(")[1].split(")")[0].strip()
        return warning
```
s/spit/split/

**Comment by tbarabosch** [comment]
thanks @gorgiaxx 
I am going to fix this ASAP. In addition, I am going to add the newer checks to the IDA Pro plugin that are not yet represented.

**Comment by tbarabosch** [comment]
This issue should be soon fixed, once PR #25 is merged.

**Comment by tbarabosch** [comment]
Fixed by #25
